bsc.typechecker/generics/generics.exp

```diff
@@ -0,0 +1,10 @@
+
+# tests for Bluespec generics
+
+compile_pass GenericTestsBSV.bsv
```

Comment:
Is there a reason that you added this line?  I don't think it's the needed -- the later command to test `GenericTests` would cause this file to be compiled (as a dependency).  If this were itself a test, then calling it out explicitly in the exp-file might make sense, but I don't think it has any meaning by itself.

Reply:
Umm, I guess I somehow didn't realize that bsc handles building dependencies - I've just been mostly working with single-file test cases.